static/app/types/debugFiles.tsx

```diff
@@ -94,17 +94,19 @@ type CustomRepoHttp = {
   layout: {casing: string; type: string};
   name: string;
   url: string;
+  username: string;
+  password: {'hidden-secret': boolean};
 };
 
 type CustomRepoS3 = {
   type: CustomRepoType.S3;
-  access_key: string;
   bucket: string;
   id: string;
   layout: {type: string; casing: string};
   name: string;
   region: string;
-  secret_key: string;
+  access_key: string;
+  secret_key: {'hidden-secret': boolean};
```

Comment:
Shouldn't this rather be:
```suggestion
  secret_key: string | {'hidden-secret': boolean};
```

Reply:
sure...according to the python code, it can also be a string...good catch...thanks

Reply:
I checked it again and it seems that the backend only sends the object back https://github.com/getsentry/sentry/pull/28192